package day39_b_exception.learn_exception;

public class SecondTry {
}
